src/app/components/home/home.component.html

```diff
@@ -15,7 +15,7 @@
 
         <ng-container *ngFor="let candidateType of sortedObj | keyvalue: asIsOrder; let i= index">
 
-          <mat-expansion-panel (opened)="setOfficeStep(i)" hideToggle="true" (closed)="setCouncilDistrictStep(-1)">
+          <mat-expansion-panel [expanded]="officeType === candidateType.key ? true : false" (opened)="setOfficeStep(i); setOfficeType(candidateType.key)" hideToggle="true" (closed)="setCouncilDistrictStep(-1)">
```

Comment:
we can shorten it, officeType===candidateType.key

Reply:
Cool, will go and shorten it.